Fall25 - M3 Milestone - Prepare r3.1 (#113)
camaraproject/OTPValidation: README.md

Reviewer: README.md comments
line 19: possessed --> possesses (real-time :-) )
line 20: 
- frequent method of verifying possession -> method for frequently verifying possession
- OTP (one-time password) --> one-time password (OTP)

In the scope: the 2 lines could be made a bit more clear:
- The One Time Password SMS API performs real-time checks to verify that the user possesses the device that carries the indicated mobile phone number, by ... ?
- It also provides a method for verifying possession of the device by delivering a one-time password (OTP) to it through SMS and validating the received code with the user's operator after the code is entered as requested.

questions: 
- - On the fist point please complete the sentence - what will the app user see or need to do in this case if any ?
On the second point I assume that the "as requested" would be part of the UI / front-end  of the application using the API ? if so it might be good to add as well.

Think about also updating the API Description on the Wiki accordingly.

## Scope

* Service APIs for “OTPValidation” (see APIBacklog.md)  
  * The OTP SMS API facilitates real-time verification of a user's device by sending a unique code via SMS to the specified mobile phone number. The process involves three key steps:
1. **Code Generation and Delivery:**  

>    The API consumer (e.g., an application) requests the operator to send an SMS containing a one-time code along with an optional message to the user's mobile device.

Reviewer: ```suggestion
   The send-code is used by the API consumer (e.g., an application) to request a one-time code to be sent by SMS to the user's mobile device along with an optional message.

```
```suggestion
   The API consumer (e.g., an application) requests the operator to send an SMS containing a one-time code along with an optional message to the user's mobile device.

```

2. **User Input:**  
   The user receives the SMS and enters the received code into the application.
3. **Verification:**  

>    The application submits the entered code back to the API, which then verifies its accuracy and responds with a confirmation indicating whether the code is valid.

Reviewer: ```suggestion
   The application submits the entered code using the validate-code API, which then verifies its accuracy and responds with a confirmation indicating whether the code is valid.

```

## Scope

* Service APIs for “OTPValidation” (see APIBacklog.md)  
  * The OTP SMS API facilitates real-time verification of a user's device by sending a unique code via SMS to the specified mobile phone number. The process involves three key steps:
1. **Code Generation and Delivery:**  

>    The API consumer (e.g., an application) requests the operator to send an SMS containing a one-time code along with an optional message to the user's mobile device.

Reviewer: The send-code API is used by the API consumer (e.g., an application) to request a one-time code to be sent by SMS to the application user's mobile phone along with ...

2. **User Input:**  
   The user receives the SMS and enters the received code into the application.
3. **Verification:**  

>    The application submits the entered code back to the API, which then verifies its accuracy and responds with a confirmation indicating whether the code is valid.

Reviewer: The application submits the entered code using the validate-code API, which ...

Reviewer: for some reason I cannot propose changes directly to the text :-(
